osm2streets/src/render.rs

```diff
@@ -491,3 +501,56 @@ fn make_sidewalk_corners(streets: &StreetNetwork, intersection: &Intersection) -
     }
     results
 }
+
+fn draw_stop_lines(
+    lane: &LaneSpec,
+    center: &PolyLine,
+    road: &Road,
+) -> Vec<(Polygon, &'static str)> {
+    let mut results = Vec::new();
+
+    if !matches!(
+        lane.lt,
+        LaneType::Driving | LaneType::Bus | LaneType::Biking
+    ) {
+        return results;
+    }
+    let thickness = Distance::meters(0.5);
+
+    let stop_line = if lane.dir == Direction::Fwd {
+        &road.stop_line_end
+    } else {
+        &road.stop_line_start
+    };
+
+    // The vehicle line
+    if let Some(dist) = stop_line.vehicle_distance {
+        if let Ok((pt, angle)) = center.dist_along(dist) {
```

Comment:
Hmm, so `dist` is along the road reference line, not even the road center line, and certainly not the lane centerline. Projecting a point to the lane center might be appropriate here. Or render one stop line for the whole road.

Eventually the projection angle could come from the intersection (because the stop lines are often parallel to the intersecting road).

Reply:
Right... for the sake of this PR, I'll keep it as it is, and think about how to make all the distances be properly along the center line, and keep that updated over time.